src/config.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"io/ioutil"
+
+	yaml "gopkg.in/yaml.v2"
+)
+
+type Config struct {
+	Host                string `yaml:"host"`
+	Port                int    `yaml:port`
+	PublicPath          string `yaml:publicpath`

```

Comment:
Why is `host` quoted but `publicpath` isn't?

Reply:
No good reason. Just how it came into existence.